manifests/00-crd-operator-config.yaml

```diff
@@ -13,6 +13,25 @@ spec:
   subresources:
     status: {}
   versions:
-  - name: v1
-    served: true
-    storage: true
+    - name: v1
```

Comment:
It may be good to do this as a separate PR.

Reply:
Can you be more specific here?  Do you want the new CRD in one PR and the unit tests in another?  What about any E2E tests (if they are added)?

Reply:
I tend to be a fan of merging smaller PRs more frequently :)

Reply:
I will break this up a little more.